webapp/src/css/font.less

```diff
@@ -6,24 +6,12 @@
   font-weight: normal;
   font-style: normal;
 }
-@font-face {
-  font-family: 'Noto';
-  src: url(/fonts/NotoSans-Italic.ttf) format("truetype");
-  font-weight: normal;
-  font-style: italic;
-}
 @font-face {
   font-family: 'Noto';
   src: url(/fonts/NotoSans-Bold.ttf) format("truetype");
   font-weight: bold;
   font-style: normal;
```

Comment:
You'll need to remove the `font-style` from here and above so Noto is used when the font-style is italic.